### ISSUE TITLE
Update skip to content to display visual button

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5948]
The first PR (https://github.com/dart-lang/site-www/pull/5894) covered an undisplayed link. Per best practices for a11y, this button should be visible. Using https://github.com/flutter/website/pull/10800 as the guideline.
